star_wars_magic_methods/people.py

```diff
@@ -1,3 +1,62 @@
 class People(object):
     def __init__(self, name, dark_side=False):
-        pass
+        self.name = name
+        self.dark_side = dark_side
+        
+        if not self.dark_side:
+            self.light_side = True
+        else:
+            self.light_side = False    
```

Comment:
instead of using `if` and `else`, you can use the `not` operator to invert the value.

`self.light_side = not self.dark_side`